lib/cms/hippo_importer/base.rb

```diff
@@ -0,0 +1,86 @@
+module Cms
+  module HippoImporter
+    class Base
+      attr_reader :records
+
+      def initialize(data:, docs: [], to: 'en', parser: HippoXmlParser)
+        @records = parser.parse(data, [hippo_type])
+        @_site = to
+        @docs = docs
+      end
+
+      def hippo_type
```

Comment:
I'd make this a constant instead. Also accessing this property from the initializer is a little strange. That's what made me do a double take.